python/fastexcel/__init__.py

```diff
@@ -165,9 +165,38 @@ def load_sheet(
                 schema_sample_rows=schema_sample_rows,
                 use_columns=use_columns,
                 dtypes=dtypes,
+                eager=False,
```

Comment:
We should probably improve the doc string here "lazy load"?
And question don't we want to have the eager version by default?

Reply:
371b2cad370db99aef35c829e27c47b7d4986260